bundles/ch.elexis.core.ui/plugin_it.properties

```diff
@@ -1,26 +1,35 @@
-CreateBills = crea fattura
+CreateBills = crea fatture
 
 Leistungen.Block         = blocco
 Leistungen.Eigenleistung = Contributo personale
 
 RechnungsManager_Standard_Fall = caso standard
 
-Textvorlagen = modelli di documento
+RefreshBillingProposal = Aggiornare la proposta di fatturazione
+
+SendMenu = Menu d/invio
+
+StartBillingProposal = Avvio della proposta di fatturazione
+
+
+Textvorlagen = Modelli del documento
 
 address = indirizzo
 
 category.description.0 = Comandi relativi alla fatturazione
 category.name.0        = elexis
 category.name.1        = banconote
 
+ch.elexis.core.ui_elexis.loginNews =  Notizie Medelexis 
+
 command.description.0   = Collegamento a un database
-command.description.1   = di nuovo all'inizio
+command.description.1   = di nuovo all/inizio
```

Comment:
Is "all/inizio" correct?

Reply:
Effectively in Italian it would be more correct " Back to the initial perspective " I inserted the fix in the code,